test/unit/org/apache/cassandra/tools/nodetool/InvalidateNetworkPermissionsCacheTest.java

```diff
@@ -46,6 +45,9 @@ public static void setup() throws Exception
 
         roleManager.createRole(AuthenticatedUser.SYSTEM_USER, ROLE_A, AuthTestUtils.getLoginRoleOptions());
         roleManager.createRole(AuthenticatedUser.SYSTEM_USER, ROLE_B, AuthTestUtils.getLoginRoleOptions());
+        roleManager.createRole(AuthenticatedUser.SYSTEM_USER, ROLE_A, AuthTestUtils.getLoginRoleOptions());
+        roleManager.createRole(AuthenticatedUser.SYSTEM_USER, ROLE_B, AuthTestUtils.getLoginRoleOptions());
```

Comment:
Do we need to add these roles twice? The test still seems to pass if I remove the duplicates...

Reply:
Not at all. Removed.